lib/plugins/particles.js

```diff
@@ -0,0 +1,7 @@
+module.exports = inject
+
+function inject (bot) {
+  bot._client.on('world_particles', (data) => {
+    bot.emit('particles', data)
```

Comment:
that's not good as @extremeheat said, there is no abstraction layer here, it is very likely already broken between versions

Reply:
In 1.8 - 1.16 only packets are changed not packets name, maybe parsing it for each version in same format will good idea.